src/Forex/ForexData.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+using Azure.Migrate.Export.Common;
+using Azure.Migrate.Export.HttpRequestHelper;
+using Azure.Migrate.Export.Models;
+
+namespace Azure.Migrate.Export.Forex
+{
+    public class ForexData
+    {
+        private static Dictionary<string, double> ExchangeRatesUSD = null;
+
+        public static double GetExchangeRate(UserInput userInputObj)
+        {
+            GetExchangeRatesFromAPI(userInputObj);
+
+            if (ExchangeRatesUSD == null ||
+                ExchangeRatesUSD.Count <= 0 ||
+                !ExchangeRatesUSD.ContainsKey(userInputObj.Currency.Key))
+            {
+                return 1.0;
+            }
+
+            return ExchangeRatesUSD[userInputObj.Currency.Key];
+        }
+
+        private static void GetExchangeRatesFromFile(UserInput userInputObj)

```

Comment:
Not needed.

Reply:
Understood, Implemented ForexData with singleton parameter.
Thanks.